### PR TITLE
Add a transformer to TinymceType to allow bbcode injection

### PR DESCRIPTION
**Pull request by itkg-ngilain**
https://github.com/open-orchestra/open-orchestra-display-bundle/pull/172
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/107
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1051
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/29

### CODE REVIEW

File: BackofficeBundle/Form/DataTransformer/BBcodeToHtmlTransformer.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+     * Take a BBcode string to turn it into html
+     * 
+     * @param string $bbCode
+     * 
+     * @return string
+     */
+    public function transform($bbCode)
+    {
+        $this->parser->parse($bbCode);
+        return $this->parser->getAsPreviewHTML();
```
Comment (itkg-alavieille) [inline BackofficeBundle/Form/DataTransformer/BBcodeToHtmlTransformer.php]: empty line

**Comment by itkg-ngilain** [comment]
@itkg-nanne, @itkg-alavieille : check this please

File: BackofficeBundle/Form/DataTransformer/BBcodeToHtmlTransformer.php (hunk 1 above)
Comment (itkg-alavieille) [inline BackofficeBundle/Form/DataTransformer/BBcodeToHtmlTransformer.php]: add an empty line

**Comment by itkg-alavieille** [comment]
:+1:  after check style and when test run

**Comment by itkg-nanne** [comment]
:+1 with line before return

**Comment by itkg-alavieille** [comment]
:+1: when test run

**Pull request merged**